.gitignore

```diff
@@ -24,7 +24,7 @@ go.work
 .env
 
 ## kzg cache
-e2e/resources/kzg/SRSTables/
- 
+../test/resources/SRSTables/
```

Comment:
regression

Reply:
curios as to why the test repo lives as an entity a level above the project directory

Reply:
Yeah not sure how that got in there.